.github/workflows/build-and-push.yml

```diff
@@ -9,11 +9,11 @@ on:
       - main
       - "jdk-.*"
 
-env:
-  cci_src_image: cimg/openjdk:11.0.20-node
-
 jobs:
   build:
+    strategy:
+      matrix:
+        jdk_version: [11.0.20, 17.0.9]
```

Comment:
Shouldn't we also include the version 8.0.312 here and merge the `jdk-8` branch (https://github.com/Jahia/cimg-mvn-cache/blob/jdk-8/.github/workflows/build-and-push.yml#L12) with this one?

Reply:
We don't build that image anymore. The existing one are enough and not need to be updated regularly